### PR TITLE
Step 4

### PR DESCRIPTION
**Pull request by CapZillator**
## What? 
Server Side Rendering implements for ArticlesList & Article components.
useMediqQuery hook replaced by useMediaQuerySSR hook.

## Why?
SEO friendly.
Improve page load time with minimized network latency.

## How?
getServerSideProps func added to ArticlesList & Article pages, where initial data parsed and transported to the corresponding page with fallback option.

### CODE REVIEW

File: src/pages/article/[id].tsx [hunk 1]
```diff
@@ -1,9 +1,37 @@
...
+import { useRouter } from 'next/router';
+import { unstable_serialize } from 'swr';
+import { articleFetcher } from 'service/articlesFetcher';
 import { Article } from 'components/article/Article';
 
 export const ArticlePage: NextPage = () => {
-  return <Article />;
+  const router = useRouter();
+  const { id } = router.query;
+  const artID = unstable_serialize(id);
```
Comment (Ilya-Valasiuk) [inline src/pages/article/[id].tsx:12]: what is artID? 
Are we working with gallery stuff? 

In our case better name would be `articleId`

File: src/pages/article/[id].tsx [hunk 2]
```diff
@@ -1,9 +1,37 @@
...
+  return <Article id={artID} />;
```
Comment (Ilya-Valasiuk) [inline src/pages/article/[id].tsx:13]: empty line before return please

File: src/pages/article/[id].tsx (hunk 1 above)
Comment (CapZillator) [reply src/pages/article/[id].tsx:12]: My bad, renamed.

File: src/pages/article/[id].tsx (hunk 2 above)
Comment (CapZillator) [reply src/pages/article/[id].tsx:13]: Formated.

**Review by Ilya-Valasiuk — APPROVED** [review]

**Pull request merged**